### ISSUE TITLE
Could you show an example to produce a key using the crate?

### ISSUE THREAD

**Issue by asmmo** [zkcrypto/bls12_381#78]

**Comment by burdges** [comment]
This is an elliptic curve crate, not a signature scheme or encryption scheme.  You'll want some other higher level crate somewhere, depending upon what your doing, but most schemes build using this curve will not be beginner friendly.

**Comment by asmmo** [comment]
yes, I'm a beginner in this field, could you suggest some crate provide an interface for signing and verifying?

**Comment by burdges** [comment]
BLS signatures are *extremely* slow unless massively aggregated.  Aggregation involves messy topics like gossip network topology.

Check out ed25519-dalek if you just need a signature scheme.  RustCrypto project has some ECDSA crates too.  Also ring.
